### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by itsMattShull**
Issue:

## What I did
Fixed spelling.

## How to test

Is this testable with jest or storyshots?

Does this need a new example in the kitchen sink apps?

Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: docs/src/pages/addons/writing-addons/index.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ export class WithNotes extends React.Component {
 
 In this case, our component can access something called the channel. It lets us communicate with the panel (where we display notes). It has a NodeJS [EventEmitter](https://nodejs.org/api/events.html) compatible API.
 
-In the above case, it will emit the notes text to the channel, so our panel can listen to it.
+In the above case, it will emit the note's text to the channel, so our panel can listen to it.
```
Comment (igor-dv) [inline docs/src/pages/addons/writing-addons/index.md]: Should it be `notes'`, since it's like a "notes addon" ?

**Review by ndelangen — APPROVED** [review]
If you can change `note's` to `notes'` it's good to go! Thanks @derekshull 👍

**Review by itsMattShull — COMMENTED** [review]
Changed `note's` to `notes'`

**Comment by derekshull** [comment]
@igor-dv Made the change to `notes'`

**Pull request merged**